puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I would love to see more options available for use in here rather than as command line things, and then a more robust template... but that seems like a "next PR" sort of thing.

At some point though, as we monitor more kinds of things, we'll need more levers in here.